### DISCUSSION TITLE
Welcome to CricketManager Discussions!

### DISCUSSION THREAD

**Discussion by arghasen** [CricketManager/CricketManager#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by patelmayur1502** [comment]
Hi wanted to know when do we expect auction or trading player feature ?
